### ISSUE TITLE
Accessibility: screen reader does not read results after typing / in editor

### ISSUE THREAD

**Issue by zersiax** [TryGhost/Ghost#10040]
### Issue Summary

On pretty much any browser and using the free, open-source screen reader NVDA, the screen reader is unable to read the results when typing / in the editor in order to go through insertable cards.
Rather, the screen reader simply reads the inserted / character, making it seem the cursor is trapped while the arrow keys are actually cycling through the cards.

### To Reproduce

1. Open the new post editor and type a / while using a screen reader. 
2. Observe that visually the up and down arrow keys, when pressed, cycle through cards whereas the screen reader only reports "/".

This is a bug, because it renders the dynamic parts of the Ghost Editor inaccessible to screen reader users. If a card has no Markdown equivalent, it cannot be used.

### Technical details:

* Presumably 2.2.4, it is the version running on the PaaS
* Node Version: Unknown
* Browser/OS: Windows 10 1809, Chrome 68
* Database: default

**Comment by kevinansfield** [comment]
This may be on interest to anyone looking to pick this issue up https://engineering.linkedin.com/blog/2018/10/making-ember-applications--ui-transitions-screen-reader-friendly

**Comment by zersiax** [comment]
I'm afraid I don't speak Ember at all, or I'd like to help. Generally though, what you'll likely need to do is populate an invisible div of some kind with an aria-live role with the results of the / command as you arrow up and down through them, this will make the screen reader read them ouut loud automatically. However, if interactions get more tricky than just a list you select an option from, you'll need to somehow add the relevant elements to the DOM and make them focusable somehow. The Monaco editor, the editor component that powers Visual Studio Code, might have some examples to work from although that is not Ember-based I'm pretty sure.
An alternative is to make the cards searchable in some kind of help reference, and to add textual tags to insert them individually the way headings and images are inserted. So, you'd essentially be adding new tags to the Markdown parser at that point. That will probably be more work, but might not require as much specific knowledge/experience.

**Comment by kevinansfield** [comment]
> An alternative is to make the cards searchable in some kind of help reference, and to add textual tags to insert them individually the way headings and images are inserted.

Cards are insertable by using the `/` command, eg. you can type `/embed` and press <kbd>Enter</kbd> to create an embed card.

**Comment by zersiax** [comment]
That already helps a lot, actually. I just tested the embed example, and the edit field to paste your embed URL seems to be accessible. Is there a list of accepted / commands/
